utils/gen-unicode-data/Sources/GenUtils/Files.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+public func readFile(_ path: String) -> String {
+  let data = FileManager.default.contents(atPath: path)!
+  return String(decoding: data, as: UTF8.self)
```

Comment:
`String(contentsOfFile:encoding:)` does this with slightly less fuss and slightly less indirection